mne/viz/ica.py

```diff
@@ -130,6 +130,256 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
     return fig
 
 
+def _create_properties_layout():
+    """creates main figure and axes layout used by plot_ica_properties"""
+    import matplotlib.pyplot as plt
+    fig = plt.figure(figsize=[7., 6.], facecolor=[0.95] * 3)
+    ax = list()
+    ax.append(fig.add_axes([0.05, 0.5, 0.3, 0.45], label='topo'))
+    ax.append(fig.add_axes([0.5, 0.6, 0.45, 0.35], label='image'))
+    ax.append(fig.add_axes([0.5, 0.5, 0.45, 0.1], label='erp'))
+    ax.append(fig.add_axes([0.05, 0.1, 0.35, 0.3], label='spectrum'))
+    ax.append(fig.add_axes([0.5, 0.1, 0.45, 0.25], label='variance'))
+    return fig, ax
+
+
+def plot_properties(inst, ica=None, picks=None, axes=None, dB=False,
+                    cmap=None, plot_std=True, sigma=None, topo_kws=None,
+                    image_kws=None, psd_kws=None, show=True):
+    """Display component properties: topography, epochs image, ERP/ERF,
+    power spectrum and epoch variance.
+
+    Parameters
+    ----------
+    inst: instance of Epochs or Raw
+        The data to use in plotting properties.
+    ica : instance of mne.preprocessing.ICA
+        The ICA solution.
+    picks : int | array-like of int | None.
+        The components to be displayed. If None, plot will show the first
+        five sources in the order as fitted. If more than one components were
+        chosen in the picks - each one will be plotted in a separate figure.
+        Defaults to None.
+    axes: list of matplotlib axes | None
+        List of five matplotlib axes to use in plotting: [topo_axis,
+        image_axis, erp_axis, spectrum_axis, variance_axis]. If None a new
+        figure with relevant axes is created. Defaults to None.
+    dB: bool
+        Whether to plot spectrum in dB. Defaults to False.
+    cmap: matplotlib colormap | None
+        Colormap to use in both topoplot and epochs image. If None topoplot
+        and epochs image use their default "RdBu_r" colormap. Defaults to None.
+    plot_std: bool | float
+        Whether to plot standard deviation in ERP/ERF and spectrum plots.
+        Defaults to True, which plots one standard deviation above/below.
+        If set to float allows to control how many standard deviations are
+        plotted. For example 2.5 will plot 2.5 standard deviation above/below.
+    sigma: float | None
+        The standard deviation of the Gaussian smoothing to apply along
+        the epoch axis in the image. If 0. or None - no smoothing is applied.
+        Defaults to None
```

Comment:
I use sigma quite often (default is 0. and I like a little smoothing) and thought it would be more convenient than passing it always in image_kws, but I can remove it, @jona-sassenhagen was not very happy about it too.